### PR TITLE
Adding new cask for DeckHub 1.0.1

### PR DESCRIPTION
**Pull request by chrismetcalf**

### CODE REVIEW

**Comment by vitorgalvao** [comment]
There was a PR template you need to fix.

**Comment by vitorgalvao** [comment]
There was a PR template you submitted. You need to fill it. It is not optional.

File: Casks/deckhub.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'deckhub' do
+  version '1.0.1'
```
Comment (vitorgalvao) [inline Casks/deckhub.rb]: Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/f25b6babcd398abf48e33af3d887b2d00de1d661/Casks/kibako.rb).

**Comment by chrismetcalf** [comment]
I'll create a new PR with the template filled and the suggested change made.

**Pull request closed without merging**